src/OSPSuite.Core/Domain/ParameterExtensions.cs

```diff
@@ -37,5 +37,14 @@ public static bool IsOfType(this IParameter parameter, PKSimBuildingBlockType bu
       {
          return parameter.BuildingBlockType.Is(buildingBlockType);
       }
+
+      /// <summary>
+      ///    Returns true if the value of the parameter was set by the user or if the formula is a constant formula or a
+      ///    distributed formula (E.g. an actual constant value that does not depend on other parameter or time)
+      /// </summary>
+      public static bool IsConstantParameter(this IParameter parameter)
```

Comment:
@Yuri05 Do you agree that this is the right definition?
In the presenter (AddAllConstant) we  do fixed and !formula.IsExplicit. This works because we filter other parameter before but this is not really ergnomic

I think constant parameters in our case are
FixedValue
Or
ConstantFormula
or DistributedFormula

Reply:
Seems correct to me. Until we find a use case where it is not :)